client/account-settings/index.js

```diff
@@ -2,12 +2,12 @@
  * External dependencies
  */
 import React from 'react';
-import { combineReducers } from 'redux';
+import { combineReducers } from 'state/utils';
```

Comment:
> What's the reason to use it, though?

Because `Calypso` has an `eslint` rule to fobbid using the regular Redux `combineReducers`. It's either use the `Calypso` one or add a bunch of `eslint` exceptions. Not a big deal either way, I just chose to comply instead of rebel :)